packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag

```diff
@@ -580,16 +615,29 @@ void main() {
   // Since we want to take shadows into account of shading instead of irradiance,
   // we had to modify the codes that multiplies the results of shadowmap into color of direct lights.
 
-  GeometricContext geometry;
+  #if THREE_VRM_THREE_REVISION >= 157
+    vec3 geometryPosition = - vViewPosition;
+    vec3 geometryViewDir = ( isOrthographic ) ? vec3( 0, 0, 1 ) : normalize( vViewPosition );
```

Comment:
I wondered why we don't have to define `geometryNormal` here, and I found that the `vec3 geometryNormal = normal` have been renamed to `nonPerturbedNormal` in r157.
For consistency, I would love if you can rename the existing `geometryNormal` in the same way and add the new `geometryNormal` here.

https://github.com/mrdoob/three.js/commit/7bf83b8b1716666958f5dd24dc49ad73111c5839#diff-0ef3a127f3d8eaf75269887ebb9da53c45aa287c14fc79b687c3dcfcd5d95cd9R74

Reply:
commit: https://github.com/pixiv/three-vrm/commit/8088de152e58a486622b9aff8664a7dc342d2af3